OptimizelySDK.Tests/OptimizelyConfigTests/OptimizelyConfigTest.cs

```diff
@@ -93,6 +93,44 @@ public void TestPollingInvalidSdkKeyGetOptimizelyConfig()
             Assert.Null(optimizelyConfigAfterDispose);
         }
 
+        [Test]
+        public void TestPollingMultipleTimesGetOptimizelyConfig()
+        {
+            HttpProjectConfigManager httpManager = new HttpProjectConfigManager.Builder()
+               .WithSdkKey("QBw9gFM8oTn7ogY9ANCC1z")
+               .WithLogger(LoggerMock.Object)
+               .WithPollingInterval(TimeSpan.FromMilliseconds(100))
+               .WithBlockingTimeoutPeriod(TimeSpan.FromMilliseconds(500))
+               .WithStartByDefault()
+               .Build(true);
+
+            Assert.NotNull(httpManager.GetConfig());
+
+            var optimizely = new Optimizely(httpManager);
+
+            var optimizelyConfig = optimizely.GetOptimizelyConfig();
+
+            Assert.NotNull(optimizelyConfig);
+            Assert.NotNull(optimizelyConfig.ExperimentsMap);
+            Assert.NotNull(optimizelyConfig.FeaturesMap);
+            Assert.NotNull(optimizelyConfig.Revision);
+
+            Thread.Sleep(210);
```

Comment:
Don't use `Thread.Sleep` add notification and use blocking collection with the timeout to block until specific revision.